addons/mail/static/src/discuss/core/public_web/discuss_patch.xml

```diff
@@ -2,7 +2,8 @@
 <templates xml:space="preserve">
     <t t-inherit="mail.Discuss" t-inherit-mode="extension">
         <xpath expr="//Thread" position="before">
-            <Call t-if="thread.rtc_session_ids.length > 0" thread="thread"/>
+            <PipBanner t-if="rtc.state.isPipMode and thread.eq(rtc.channel)"/>
+            <Call t-elif="thread.rtc_session_ids.length > 0" thread="thread"/>
```

Comment:
Be cautious that `>` is a risk of XML parser that can misread it for closing of tag name node. We should normally use `gt` that is replaced by `>` during code evaluation of the template.